src/screens/Header/Nav.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Nav from '../../components/Header/Nav/Nav';
+
+const ScreensHeaderNav = () => {
```

Comment:
not sure if this should be the screen, because you don't have `headerNav` route. I think that it should be in the components folder